tests/samples.ts

```diff
@@ -215,3 +139,118 @@ export const albMultiValHeadersRequest = (): ApplicationLoadBalancerRequestEvent
       },
    });
 };
+
+export const makeAPIGatewayRequestContextIdentity = (): APIGatewayEventIdentity => {
+   return {
+      accessKey: null,
+      accountId: null,
+      apiKey: null,
+      apiKeyId: null,
+      caller: null,
+      clientCert: null,
+      cognitoAuthenticationProvider: null,
+      cognitoAuthenticationType: null,
+      cognitoIdentityId: null,
+      cognitoIdentityPoolId: null,
+      principalOrgId: null,
+      sourceIp: '12.12.12.12',
+      user: null,
+      userAgent: 'curl/7.54.0',
+      userArn: null,
+   };
+};
+
+export const makeAPIGatewayRequestContext = (params?: MakeAPIGatewayRequestEventContextParams): APIGatewayEventRequestContext => {
+   return {
+      accountId: '123456789012',
+      apiId: 'someapi',
+      authorizer: null,
+      httpMethod: params?.httpMethod ?? 'GET',
+      path: '/prd',
+      protocol: 'HTTP/1.1',
+      stage: 'prd',
+      requestId: 'a507736b-259e-11e9-8fcf-4f1f08c4591e',
+      requestTimeEpoch: 1548969891530,
+      resourceId: 'reas23acc',
+      identity: makeAPIGatewayRequestContextIdentity(),
+      resourcePath: '/',
+   };
+};
+
+export const makeAPIGatewayRequestEventHeaders =
+   (params?: MakeAPIGatewayRequestEventHeadersParams): APIGatewayRequestEvent['headers'] => {
```

Comment:
For functions, we prefer the `function` declaration style. The other functions in this file were from before we had a set standard.
```suggestion
export function makeAPIGatewayRequestEventHeaders(params?: MakeAPIGatewayRequestEventHeadersParams): APIGatewayRequestEvent['headers'] {
```

Reply:
@onebytegone should I refactor all functions from this file for consistency or just the new ones?